library/std/Cargo.toml

```diff
@@ -18,7 +18,8 @@ panic_unwind = { path = "../panic_unwind", optional = true }
 panic_abort = { path = "../panic_abort" }
 core = { path = "../core", public = true }
 libc = { version = "0.2.153", default-features = false, features = ['rustc-dep-of-std'], public = true }
-compiler_builtins = { version = "0.1.105" }
+# FIXME: 0.1.106 causes issues in the build script
+compiler_builtins = { version = "=0.1.105" }
```

Comment:
Note that #120055 wants to update this to 0.1.108.

Reply:
If that lands okay I'll rebase and bump this up